Resources/Private/Templates/MusiconnPublish/MusiconnHome.html

```diff
@@ -0,0 +1,19 @@
+<html data-namespace-typo3-fluid="true" xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers">
+
+<f:layout name="Default" />
+
+<f:section name="Main">
+    <div class="container homepage">
+        <h1 class="sr-only">{pageTitle}</h1>
+        <section class="home-search">
+            <f:cObject typoscriptObjectPath="lib.dynamicContent" data="{colPos: '11'}" />
+            <f:cObject typoscriptObjectPath="lib.navsearch" />
```

Comment:
Ich bin mir hier nicht sicher. In der DDEV-Umgebung und im Demo-System überlagert der Suchschlitz die zwei Inhaltselemente daneben, durch seinen grauen Rahmen. Entweder die zwei Inhaltselemente kommen aus der `home-search`-section heraus oder die Suche braucht noch mehr `padding` im CSS.

![image](https://github.com/slub/slub_web_qucosa/assets/180686/dac6e903-3e76-4231-8043-fe6e80d41a6e)

Reply:
Aufgrund veränderter Klassen-Namen ging das leider etwas schief. Ist nun gefixt.